pkg/pb/node.proto

```diff
@@ -14,20 +14,22 @@ message NodeRestrictions {
     int64 free_disk = 2;
 }
 
+// TODO move statdb.Update() stuff out of here
```

Comment:
why does it have to get moved out?

Reply:
it's cluttering the node protobuf that is used for components besides statdb, and those variables are only used for like one statdb function. We can get rid of them by just changing how we use statdb in the audit package in the future. We draw a lot of unnecessary functionality out of statdb.Update()